### DISCUSSION TITLE
Where is data stored?

### DISCUSSION THREAD

**Discussion by blairwilcox** [six7/figma-tokens#60] [Q&A] [answered]
Hi everyone!

This plugin is great and solves a lot of problems my team was having, so thank you! One thing I'm wondering about is how the design token data is currently stored? We maintain a design system that's used across our organization, so we'd like to be able to have a common source everywhere. Is this currently supported? And if not, could you explain a bit about how token-storage works so we can add this functionality ourselves?

**Comment by six7** [comment, marked as answer]
Hey! Awesome, glad it's of use to you!

I do have plans for token storage and sync, however I'm currently thinking of making that a paid feature of the plugin. The main advantage would be that you'd have a website where you could edit your tokens as well as edit them in the plugin, plus the huge benefit of having one source for the whole team.

Currently all tokens are stored in the file itself via sharedPluginData, which is shared between different people using the file.

I dont have a timeline for external sync yet, but it's one of  the next steps for the plugin (next to Export).

**Reply by six7** [reply]
FYI: Token sync is now enabled!
